bin/check-marathon-apps.rb

```diff
@@ -0,0 +1,398 @@
+    |
+    |# Publish '"aggregate": "component"' field in the check results for all health and status checks.
+    |- SENSU_MARATHON_AGGREGATE=component
+    |# similar for some other fields
+    |- SENSU_MARATHON_CONTACT=support
+    |
+    |# Publish '"aggregate": "component"' field in the check results only for status checks.
+    |- SENSU_MARATHON_STATUS_AGGREGATE=component
+    |# Don't handle status check results
+    |- SENSU_MARATHON_STATUS_HANDLE=false
+    |
+    |# Generate UNKNOWN result for unknown health status, rather then the default OK result
+    |- SENSU_MARATHON_HEALTH_UNKNOWN_STATUS=3
+    |
+    |Similar logic could be applied for the provided default_check_config. Values
+    |under the special '_' key will be used as a default value for deeper levels.
+    |
+    |Here is the default value for default_check_config:
+    |#{DEFAULT_CHECK_CONFIG}
+    |
+    |Options:
+  BANNER
+
+  option :url,
+         description: 'Marathon API URL',
+         short: '-u url',
+         long: '--url url',
+         default: 'http://localhost:8080'
+
+  option :username,
+         short: '-u USERNAME',
+         long: '--username USERNAME',
+         default: nil,
+         required: false,
+         description: 'Marathon API username'
+
+  option :password,
+         long: '--password PASSWORD',
+         default: nil,
+         required: false,
+         description: 'Marathon API password'
+
+  option :match_pattern,
+         short: '-m PATTERN',
+         long: '--match-pattern PATTERN',
+         required: false,
+         description: 'Match app names against a pattern, exlude pattern takes precedence if both provided'
+
+  option :exclude_pattern,
+         short: '-x PATTERN',
+         long: '--exclude-pattern PATTERN',
+         required: false,
+         description: 'Exclude apps that match a pattern, takes precedence over match pattern'
+
+  option :marathon_keys,
+         long: '--marathon-keys KEY1,KEY2,KEY3',
+         default: 'id,version,versionInfo,tasksStaged,tasksRunning,tasksHealthy,tasksUnhealthy,lastTaskFailure',
+         required: false,
+         description: 'Keys retrieved from Marathon API that will be included in the output'
+
+  option :default_check_config,
+         long: '--default-check-config "{"status":{"running":{"valid":"json"}},"health":{"healthy":{"valid":"json"}}}"',
+         required: false,
+         description: 'Default values to be used while creating the check results, '\
+                      'can be overridden in a per-marathon application config via Marathon labels.'
+
+  option :default_check_config_file,
+         long: '--default-check-config-file CONFIG_FILE',
+         required: false,
+         description: 'Similar to `--default-check-config` but read from given file. If both parameters are provided  '\
+                      '`--default-check-config` will override this one.'
+
+  option :sensu_client_url,
```

Comment:
would this be more appropriate to call this `sensu_client_socket`?

Reply:
Using the term `socket` rather reminds me the unix sockets, this one is actually a url with an actual scheme (`http`), but i'm removing the `socket` keyword from the options description, that's misleading.